tests/source/issue-4222.rs

```diff
@@ -0,0 +1,6 @@
+fn first_function() {
+    struct inner_item {
+        field: u32,
+    }
```

Comment:
We should add a trailing semicolon here to make sure that rustfmt removes it.

```suggestion
    };
```

Reply:
haha yeah, that's important... thanks!